scripts/build/release_publisher/publisher.go

```diff
@@ -174,6 +174,11 @@ var completeBuildArtifactConfigurations = []buildArtifact{
 		arch:       "amd64",
 		urlPostfix: ".windows-amd64.zip",
 	},
+	{
+		os:         "win",
```

Comment:
This looks wrong to me, looking at the MSI support PR https://github.com/raintank/grafana-net/pull/736/files#diff-2f99f3c6f7e16e0179ad7a0809319427R29  the OS should be `win-installer`